### PR TITLE
The react warnings

### PR DESCRIPTION
**Pull request by jamesli898**
we have fixed the react warnings on the app.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
     "pg": "^7.12.1",
     "react": "^16.10.2",
     "react-dom": "^16.10.2",
-    "react-router-dom": "^5.1.2",
+    "react-router-dom": "^4.4.0-beta.6",
```
Comment (abaruah117) [inline package.json]: pls fix

Comment (abaruah117) [inline package.json]: pls revert this

File: src/index.js [hunk 2]
```diff
@@ -16,3 +16,5 @@ ReactDOM.render(
 // unregister() to register() below. Note this comes with some pitfalls.
 // Learn more about service workers: https://bit.ly/CRA-PWA
 serviceWorker.unregister();
+
+export default BrowserRouter;
```
Comment (abaruah117) [inline src/index.js]: why is this here?

**Review by abaruah117 — CHANGES_REQUESTED** [review]
small comments

File: src/index.js (hunk 2 above)
Comment (jamesli898) [reply src/index.js]: fixed

**Review by abaruah117 — APPROVED** [review]
lgtm
